woocommerce/woocommerce
# [POC] Enforce style on express checkout buttons

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:






P2 Post: pdFofs-1O1-p2

This is POC PR to demonstrate limited control over the express checkout buttons using the wrapper component. 

In this PR, we're enforcing the style on express checkout buttons. If there are two express checkout buttons, it will display them in a single column instead of the single row. 

The branch is based on https://github.com/woocommerce/woocommerce/pull/44346 PR branch. 



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Activate the Stripe plugin and enable express payment options. 
2. Activate the PayPal payment plugin and enable express payment options. 
3. Add a product to the Cart and visit the Checkout block page with `https`.
4. Confirm the below express payment buttons is displayed in a single column:
![image](https://github.com/woocommerce/woocommerce/assets/11503784/52ecb303-895c-4878-81ed-441068a24454)




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
